Libs/MRML/DisplayableManager/vtkMRMLAbstractSliceViewDisplayableManager.h

```diff
@@ -58,21 +58,25 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLAbstractSliceViewDisplayableMana
   static void ConvertDeviceToXYZ(vtkRenderer* renderer, vtkMRMLSliceNode* sliceNode, double x, double y, double xyz[3]);
 
   /// Convert RAS to XYZ coordinates (viewport).
-  /// Parameters \a ras and \a xyz are double[3]. \a xyz[2] is the lightbox id.
+  /// Parameters \a ras and \a xyz are double[3]. \a xyz[2] used to be the lightbox id
+  /// and is now always set to 0.0.
   /// \sa ConvertRASToXYZ(vtkMRMLSliceNode* sliceNode, double ras[3], double xyz[3])
   void ConvertRASToXYZ(double ras[3], double xyz[3]);
 
   /// Convenience function allowing to convert RAS to XYZ coordinates (viewport).
-  /// Parameters \a ras and \a xyz are double[3]. \a xyz[2] is the lightbox id.
+  /// Parameters \a ras and \a xyz are double[3]. \a xyz[2] used to be the lightbox id
+  /// and is now always set to 0.0.
```

Comment:
Should the ConvertRASToXYZ be deprecated and instead introduce a ConvertRASToXY to avoid confusion about what the Z would be ?